docs/installation/additional_integration.asciidoc

```diff
@@ -353,46 +353,48 @@ Logs will be kept on PacketFence **and** sent to your remote Syslog server.
 
 For further reference the monit documentation is available at: https://mmonit.com/monit/documentation/monit.html
 
-==== Install Monit
+Monit configuration path is different between EL and Debian systems:
 
-The following must be done on each server of a cluster
+* [filename]`MONIT_PATH=/etc/monit.d` for EL-based systems
+* [filename]`MONIT_PATH=/etc/monit/conf.d` for Debian-based systems
 
-RHEL / CentOS
+To simplify further documentation, we will use `$MONIT_PATH` variable as a reference to these paths.
 
-----
-yum install monit --enablerepo=packetfence-extra -y
-----
+==== Install Monit
 
-Debian
+The following must be done on each server of a cluster
 
+.RHEL / CentOS
+[source,bash]
 ----
-apt-get update
-apt-get install monit
+yum install monit uuid --enablerepo=packetfence -y
 ----
 
-
-==== Fetch the script signing key
-
+.Debian
+[source,bash]
 ----
-gpg --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys E3A28334
+apt-get update
+apt-get install monit uuid-runtime
 ----
 
+===== Enable httpd interface (Debian-based systems only)
```

Comment:
Shouldn't we simply include that in the files we generate to enable it so that users don't have to do this manual step ?

Reply:
I didn't do it because it's already enabled by default on EL systems.
I can create an issue to handle that later.